packages/retag/src/index.ts

```diff
@@ -139,8 +139,8 @@ export async function fetchTypesPackageVersionInfo(
   if (info.deprecated) {
     // https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22306
     assert(
-      pkg.name === "angular-ui-router" || pkg.name === "ui-router-extras",
-      `Package ${pkg.name} has been deprecated, so we shouldn't have parsed it. Was it re-added?`
+      pkg.typesDirectoryName === "angular-ui-router" || pkg.typesDirectoryName === "ui-router-extras",
```

Comment:
Not sure if `typesDirectoryName` includes a version or not, but maybe this should just be checking `=== "@types/angular-ui-router"`? Not sure what this check is doing at this point though.